src/test/indexDom.test.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @param {boolean} add - If true, increments the pageCount by 1.
+ * @param {boolean} sub - If true, decrements the pageCount by 1.
+ */
+
+let pageCount = 0
+
+function moveMenu (add, sub) {
+  if (add) {
+    pageCount += 1
+  }
+  if (sub) {
+    pageCount -= 1
+  }
+  if (pageCount < 0) {
+    pageCount = 0
+  }
+}
+
+describe('Highlight menu count', () => {
+  test('Check if count starts at 0', () => {
+    expect(pageCount).toBe(0)
+  })
+
+  test('Check increasing count', () => {
+    moveMenu(true, false)
+    expect(pageCount).toBe(+1)
+  })
+
+  test('Check decreaseing count', () => {
+    moveMenu(false, true)
+    expect(pageCount).toBe(0)
+  })
+
+  test('Ensure nothing else can affect count', () => {
+    moveMenu(false, false)
+    expect(pageCount).toBe(0)
+  })
```

Comment:
It seems that there is one case missing: true + true

Reply:
Yes, moving the function outside îs going to be the next step :)

Reply:
@Alexandru-Bulai have you made any changes regarding this?

Reply:
@vladiiancu  Yessir, i added the case on the last Commit is im not mistaking